variant-inc/terraform-aws-sns
# Feature/cloud 1484 sns module

# Description

Terraform module to create sns topic.

Fixes [#2 ](https://usxtech.atlassian.net/browse/CLOUD-1484)

## Type of change


## How Has This Been Tested?

```yaml
Pre reqs:
  - Clone/Pull branch
  - Create terraform.tfvars.json 
{
    "aws_resource_name_prefix":"dpl-",
    "name": "test-topic-naveen",
    "kms_key_sns_alias_arn":"arn:aws:kms:us-east-1:786352483360:key/255fb0cb-ac0b-4237-acf8-93ffd62c988e"
}

  - Create provider.tf (provided in readme.md)

Test Case 1 [Create SNS topic]:
- Steps
  - Terraform init
  - Terraform plan/apply
- Result
 - Should create sns topic

Test Case 2 [Destroy Sns topic]:
- Steps
  - terraform destroy
- Results
  - See Bucket deleted
```

## Checklist
